README.md

```diff
@@ -72,3 +80,21 @@ We use jenkins to deploy. Make sure you can access [https://jenkins.eanadev.org/
 
 - `statistics-dashboard-test`
 - `statistics-dashboard-acceptance`
+
+## Docker
+
+To make a (parameterised) docker image of the app first run this command:
+
+`npm run dist-localised`
+
+and then copy the output of that command (from the dist directory) into a docker nginx image:
+
+`docker build -t statistics-dashboard-app-image:v2.0 .`
+
+The image runtime configuration is only set on container startup, so environment variables have to be passed when it's run.  Here they are passed using the project `env_file`:
+
+`docker run -it --rm -d -p 80:80  --env-file=env_file --name stats-db statistics-dashboard-app-image:v2.0`
```

Comment:
It gives me an error when I try with this port, perhaps you meant 8080:8080?
I had to write a different port to work

Reply:
If you already have `nginx` running then it will block, but it can indeed map to another port number!